sdk/digitaltwins/azure-digitaltwins-core/src/main/java/module-info.java

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+module come.azure.digitaltwins.core {
+    requires transitive com.azure.core;
+    requires com.fasterxml.jackson.annotation;
+    requires com.azure.http.netty;
```

Comment:
Jackson should be available transitively through azure core and netty should not be included in this list. Http clients are pluggable and netty can be replaced by another http client.